studio/components/interfaces/Support/SupportForm.tsx

```diff
@@ -75,6 +76,12 @@ const SupportForm: FC<Props> = ({ setSentCategory }) => {
     }
   }, [uploadedFiles])
 
+  useEffect(() => {
+    get(`${API_URL}/profile`).then((data) => {
```

Comment:
update to use the `useProfileQuery`, which will remove the need for a state + useEffect